torchaudio/utils/sox_utils.py

```diff
@@ -0,0 +1,84 @@
+from typing import List, Dict
+
+import torch
+
+from torchaudio._internal import (
+    module_utils as _mod_utils,
+)
+
+
+@_mod_utils.requires_module('torchaudio._torchaudio')
+def set_seed(seed: int):
+    """Set libsox's PRNG
+
+    Args:
+        seed: seed value. valid range is int32.
+
+    See Also:
+        http://sox.sourceforge.net/sox.html
+    """
+    torch.ops.torchaudio.sox_utils_set_seed(seed)
+
+
+@_mod_utils.requires_module('torchaudio._torchaudio')
+def set_verbosity(verbosity: int):
+    """Set libsox's verbosity
+
+    Args:
+        verbosity: Set verbosity level of libsox.
+            1: failure messages
+            2: warnings
+            3: details of processing
+            4-6: increasing levels of debug messages
+
+    See Also:
+        http://sox.sourceforge.net/sox.html
+    """
+    torch.ops.torchaudio.sox_utils_set_verbosity(verbosity)
+
+
+@_mod_utils.requires_module('torchaudio._torchaudio')
+def set_buffer_size(buffer_size: int):
```

Comment:
When I see functions like these, I understand that there are global parameters behind the scene to manage the `sox_effects_chain`. Is that the case? Is there a way here to have many chains with different such settings?

Reply:
> When I see functions like these, I understand that there are global parameters behind the scene to manage the `sox_effects_chain`. Is that the case? 

Yes

> Is there a way here to have many chains with different such settings?

Nope